DynamicPackagingVODContent/StreamHLSAndDASH/src/main/java/sample/StreamHLSAndDASH.java

```diff
@@ -123,57 +131,68 @@ private static void runStreamHlsAndDash(ConfigWrapper config) {
                 System.out.println();
 
                 // Now that the content has been encoded, publish it for Streaming by creating
-                // a StreamingLocator. 
-                StreamingLocator locator = getStreamingLocator(manager, config.getResourceGroup(), config.getAccountName(),
+                // a StreamingLocator.
+                StreamingLocator locator = getStreamingLocator(manager,
+                        config.getResourceGroup(), config.getAccountName(),
                         outputAsset.name(), locatorName);
 
                 StreamingEndpoint streamingEndpoint = manager.streamingEndpoints()
-                        .getAsync(config.getResourceGroup(), config.getAccountName(), STREAMING_ENDPOINT_NAME)
-                        .toBlocking().first();
+                        .get(config.getResourceGroup(), config.getAccountName(),
+                                STREAMING_ENDPOINT_NAME);
 
                 if (streamingEndpoint != null) {
                     // Start The Streaming Endpoint if it is not running.
                     if (streamingEndpoint.resourceState() != StreamingEndpointResourceState.RUNNING) {
-                        manager.streamingEndpoints().startAsync(config.getResourceGroup(), config.getAccountName(), STREAMING_ENDPOINT_NAME).await();
+                        manager.streamingEndpoints().start(config.getResourceGroup(),
+                                config.getAccountName(), STREAMING_ENDPOINT_NAME);
 
                         // We started the endpoint, we should stop it in cleanup.
                         stopEndpoint = true;
                     }
 
-                    List<String> urls = getHlsAndDashStreamingUrls(manager, config.getResourceGroup(), config.getAccountName(), locator.name(), streamingEndpoint);
+                    List<String> urls = getHlsAndDashStreamingUrls(manager,
+                            config.getResourceGroup(), config.getAccountName(), locator.name(),
+                            streamingEndpoint);
                     System.out.println();
                     for (String url : urls) {
                         System.out.println(url);
                     }
                     System.out.println();
 
                     System.out.println();
-                    System.out.println("Copy and paste the Streaming URL into the Azure Media Player at 'http://aka.ms/azuremediaplayer'.");
+                    System.out.println(
+                            "Copy and paste the Streaming URL into the Azure Media Player at 'http://aka.ms/azuremediaplayer'.");
```

Comment:
We can also build the URL above and print it out so that customer can click and launch the AMP player with the URL directly. I did that in many other samples.

Reply:
how can we construct urls for VOD? what will be the manifest name?